.github/workflows/delete-merged-branches.yml

```diff
@@ -0,0 +1,26 @@
+name: Branch Cleanup
+
+on:
+  pull_request:
```

Comment:
can we run action on merge?

Reply:
@TanishqSingla could you check this out.